.github/workflows/ubuntu.yml

```diff
@@ -28,7 +28,8 @@ jobs:
           libglew-dev \
           libglfw3-dev \
           libwayland-dev \
-          libxnvctrl-dev
+          libxnvctrl-dev \
+          libxkbcommon-dev
```

Comment:
This dep should be added to [build_deps.sh](https://github.com/flightlessmango/MangoHud/blob/master/build_deps.sh)

arch - `libxkbcommon`
fedora - `libxkbcommon-devel`
solus - `libxkbcommon-devel`
suse - `libxkbcommon-devel`